src/libs/datatableproviders/DataProvider.ts

```diff
@@ -0,0 +1,78 @@
+// define metadata structures
+interface EntityTypeMetadata {
```

Comment:
I'm surprised that an indent size of 4 is allowed by the linter.

Reply:
I wonder if [this line](https://github.com/DataBiosphere/terra-ui/blob/2ada267429b274e8ecaa8aea1d242033cef450a3/.eslintrc.js#L121) means the linter isn't examining the `.ts` files.

Reply:
@kyuksel my VSCode shows linting issues without changing that line, and `yarn lint` fixes problems in `DataProvider.ts` … so I have not changed that line.

@cahrens I've changed to 2 spaces!